autoload/direnv.vim

```diff
@@ -5,6 +5,7 @@
 scriptencoding utf-8
 
 let s:direnv_cmd = get(g:, 'direnv_cmd', 'direnv')
+let s:direnv_interval = get(g:, 'direnv_interval', 500)
 let s:direnv_auto = get(g:, 'direnv_auto', 1)
 let s:job_status = { 'running': 0, 'stdout': [], 'stderr': [] }
```

Comment:
You removed `running` from `direnv#job_status_reset()`, but not here.

Reply:
Yes. We need it.